src/csharp/Microsoft.Spark/SparkContext.cs

```diff
@@ -299,6 +304,24 @@ public void SetCheckpointDir(string directory)
             _jvmObject.Invoke("setCheckpointDir", directory);
         }
 
+        /// <summary>
+        /// Broadcast a read-only variable to the cluster, returning a Microsoft.Spark.Broadcast
+        /// object for reading it in distributed functions. The variable will be sent to each 
+        /// executor only once.
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <param name="value">Value/variable to be broadcast</param>
+        /// <returns>A Microsoft.Spark.Broadcast object</returns>
+        public Broadcast Broadcast<T>(T value)
+        {
+            var local_dir = (string)_jvmObject.Jvm.CallStaticJavaMethod(
```

Comment:
Please check the coding guideline for var usage.

Reply:
I think the usage of `var` here is correct.